.github/workflows/ci.yml

```diff
@@ -197,18 +197,16 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2
         with:
-          ref: ${{ github.event.after }}  # for PR avoids checking out merge commit
           fetch-depth: 0  # include all history
 
+      - name: "Set GITHUB_BRANCH"
+        run: |
```

Comment:
we should not change this file, if we need changes we need to add to template_config.ymlz there is a ci_env there to add additional environment variables.

If something needs fix then we have to fix on https://github.com/pulp/plugin_template

we need to use plugin_template to generate some of the CI files

Reply:
update manifest is on .github/post_job_templatwz you can edit that file and then run plugin_template

Reply:
https://github.com/ansible/galaxy_ng/blob/master/.github/post-job-template.yml.j2

Reply:
Aah, thanks!

Updated that file, `./plugin-template --github galaxy_ng` no longer complains (the j2 syntax clash with github's `${{` is unfortunate, and leads to a rather cryptic `jinja2.exceptions.UndefinedError: 'github' is undefined`), and `ci.yml` stays the same :+1: